tagger/firmware/hls4ml_convert.py

```diff
@@ -45,7 +45,7 @@ def convert(model, outpath,build=True):
 
     
     config["LayerName"]["jet_id_output"]["Precision"]["result"] = class_precision
-    config["LayerName"]["jet_id_output"]["Implementation"] = "legacy"
+    config["LayerName"]["jet_id_output"]["Implementation"] = "latency"
```

Comment:
Are the synthesized results saved somewhere? I guess it can be downloaded as a CERN Gitlab artifact for now?